qiskit_ibm_provider/ibm_provider.py

```diff
@@ -416,6 +416,9 @@ def active_account(self) -> Optional[Dict[str, str]]:
         Returns:
             A dictionary with information about the account currently in the session.
         """
+        if not self._account.instance:
+            hgp = self._get_hgp()
+            self._account.instance = f"{hgp._hub}/{hgp._group}/{hgp._project}"
```

Comment:
`self._account.instance` is never updated, even after a job is run - if a user wants to see the default hgp by calling `active_account()` we can use `_get_hgp()` which returns the default hgp

This doesn't seem like the cleanest solution but I couldn't find a better way

Reply:
First hgp in the list of hgps will always be the one that is selected by default.
We need to always return this no matter what is already in self._account.
So something like below should work, please test it:
```
hgps = self._get_hgps()
active_account_dict = self._account.to_saved_format()
active_account_dict.update({ "instance": hgps[0].name })
return active_account_dict
```

Reply:
Tested, works as intended